src/plugins/data/common/search/aggs/buckets/_interval_options.ts

```diff
@@ -39,48 +39,48 @@ export const intervalOptions = [
     display: i18n.translate('data.search.aggs.buckets.intervalOptions.millisecondDisplayName', {
       defaultMessage: 'Millisecond',
     }),
-    val: 'ms',
```

Comment:
why was it changed? Does this mean that the user cannot set the `ms` value from the keyboard?

Reply:
=(
![image](https://user-images.githubusercontent.com/20072247/98973522-e8388280-2524-11eb-8cba-49892f112671.png)

Reply:
fixed